frontend/index.html

```diff
@@ -1,23 +1,14 @@
 <!doctype html>
-<html lang="eng">
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Group Income</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
     <link rel="stylesheet" href="/assets/css/main.css">
     <link rel="icon" href="/assets/images/group-income-icon-transparent.png" sizes="32x32">
   </head>
   <body lang="eng">
```

Comment:
Remove `lang` from `body`, as it's an invalid value (and redundant)

Reply:
Good catch - sorry for overlooking it